components/application-registry/README.md

```diff
@@ -31,8 +31,10 @@ The Application Registry has the following parameters:
 - **externalAPIPort** - This port exposes the Metadata API to an external solution. The default port is `8081`.
 - **minioURL** - The URL of a Minio service which stores specifications and documentation.
 - **namespace** - Namespace where Application Registry is deployed. The default Namespace is `kyma-system`.
-- **requestTimeout** - A time-out for requests sent through the Application Registry. It is provided in seconds. The default time-out is `1`.
+- **requestTimeout** - A timeout for requests sent through the Application Registry. It is provided in seconds. The default timeout is `1`.
```

Comment:
```suggestion
- **requestTimeout** - A time-out for requests sent through the Application Registry. It is provided in seconds. The default time-out is `1`.
```

Reply:
Because we follow [MSG](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/time-out) ;)

Reply:
Done